ci/run-tests.sh

```diff
@@ -5,7 +5,6 @@
 cd $(dirname "$(readlink -f "$0")")/..
 printenv | sort
 mvn -B -V install -DskipTests -Dmaven.javadoc.skip=true
-jabba use ${TEST_JAVA_VERSION}
```

Comment:
This was required by CCM right?

Reply:
This change should be reverted together with the CcmBridge changes when your PR-2000 is merged.